packages/components/src/flyout/types.ts

```diff
@@ -11,7 +11,7 @@ import type { CSSProperties, FunctionComponentElement } from 'react';
  */
 import type { PopperProps } from '../utils/types';
 
-export type PopoverContext = {
+export type Context = {
 	popover?: PopoverStateReturn;
```

Comment:
Should we also rename `popover` to `flyout` (or `flyoutState`)?

Reply:
Yeah! I think that makes sense.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32197/commits/8bd3af8656f462cafd26871e439d38ca53bd6eae